### PR TITLE
QUIC is QUIC, and does not stand for anything.

### PR DESCRIPTION
**Pull request by ariel-anieli**
https://www.ietf.org/mail-archive/web/quic/current/msg03855.html

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -137,6 +137,9 @@ protocol, including the conceptual design, wire format, and mechanisms of the
 QUIC protocol for connection establishment, stream multiplexing, stream and
 connection-level flow control, connection migration, and data reliability.
 
+QUIC used to be known as 'Quick UDP  Internet Connections', but the present
+document advises it be used no  more: QUIC does not stand for anything.
+
 Accompanying documents describe QUIC's loss detection and congestion control
```
Comment (mikkelfj) [inline draft-ietf-quic-transport.md]: I belive the feedback on the mailing list was that the RFC is not the place to make historical references.

>QUIC used to be known as 'Quick UDP  Internet Connections', but the present
 +document advises it be used no  more: QUIC does not stand for anything.

could perhaps be reformulated as

> QUIC is a name and not an acronym.

**Comment by ariel-anieli** [comment]
@mikkelfj, here is the new pull.

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (ariel-anieli) [reply draft-ietf-quic-transport.md]: Fixed in the new pull, @mikkelfj. Have you seen it?

Comment (mikkelfj) [reply draft-ietf-quic-transport.md]: now I have :)

**Comment by MikeBishop** [comment]
I'm a little dubious of throwing this sentence as a stand-alone paragraph in the Introduction, as it seems unrelated to most of the other text there.  Maybe it would fit better at the end of Section 2, when we're defining other terms and notations?

**Comment by ariel-anieli** [comment]
Right, @MikeBishop. A new pull will change it.

**Comment by martinthomson** [comment]
Thanks Mike, I had the same objection, but didn't think of the suggestion.

**Comment by ariel-anieli** [comment]
Fixed, @MikeBishop.

**Comment by martinthomson** [comment]
So few words, so much effort.  Thanks @ariel-anieli.

**Pull request merged**